chain-connect/src/GalachainConnectClient.ts

```diff
@@ -26,12 +26,16 @@ export class GalachainConnectClient {
   #provider: BrowserProvider | undefined;
   #chainCodeUrl: string;
 
-  get address() {
+  get ethAddress() {
```

Comment:
looks like these got mixed up, this is retuning the galachain address and the other one is returning the eth address. Given what you use in the setter, this should just be `return this.#address`